include/boost/histogram/detail/nonmember_container_access.hpp

```diff
@@ -45,6 +45,12 @@ constexpr E* data(std::valarray<E>& v) noexcept {
   return std::begin(v);
 }
 
+#if __cpp_lib_nonmember_container_access >= 201411L
```

Comment:
This is a cosmetic change, please revert.

Reply:
Got it. Reverted.

---

(The original intent of this change is to use unqualified `size` inside `include\boost\histogram\detail\eytzinger_search.hpp` to let `size` resolve to user provided one (if there is one) through adl.)

- Before C++17 there is no `std::size` so I `#include<boost/histogram/detail/nonmember_container_access.hpp>`.
- From C++17 `std::size` conflicts with `detail::size` **when I write unqualified `size`** so I added `#if`.

I checked the doc just now. Seems the `size` is intended to be a customization point, so if I use `detail::size` then there is no adl then the customizability of `size` will be lost.
https://en.cppreference.com/w/cpp/iterator/size says:
> Custom overloads of size may be provided for classes and enumerations that do not expose a suitable size() member function, yet can be detected.